### DISCUSSION TITLE
OpenTelemetry not showing database queries for tomcat application

### DISCUSSION THREAD

**Discussion by Dark002** [open-telemetry/opentelemetry-java-instrumentation#9954] [Q&A]
Hi, I have a spring application based on gradle which has multiple modules. I am using opentelemetry to get traces of my app. When I deploy a module not based on tomcat, it runs perffectly showing all traces, including database names. But when I deploy tomcat based module, it only shows the function called, not the database name. I am using runtime attachment to attach opentelemetry:  **_RuntimeAttach.attachJavaagentToCurrentJvm()_**. Can any body please tell me how to resolve this issue? I have looked into the documentation but couldn't find any solution.

**Comment by laurit** [comment]
@Dark002 Please provide a minimal sample application that reproduces the issue.
